app/src/routes/$id.inntekt-og-refusjon.tsx

```diff
@@ -90,7 +99,17 @@ function InntektOgRefusjon() {
       naturalytelserSomMistes:
         inntektsmeldingSkjemaState.naturalytelserSomMistes.length === 0
           ? [NATURALYTELSE_SOM_MISTES_TEMPLATE]
-          : inntektsmeldingSkjemaState.naturalytelserSomMistes,
+          : inntektsmeldingSkjemaState.naturalytelserSomMistes.map(
```

Comment:
Denne er jeg ikke helt fornøyd med. Nå når det måtte konverteres mellom boolean til JaNei dypt nedi ble det rotete synes jeg. Tenker den kan leve slik for nå, men at vi bør finne penere måte å gå mellom de to datamodellene våre på?

Reply:
Hva er egentlig businesslogikken her? Hvorfor trenger vi dette "inkluderTom"-feltet? I have so many questions.

Reply:
[Her er en tråd med litt kontekst her og der](https://nav-it.slack.com/archives/C04HJAAMGKC/p1724921327300729)

Kort oppsummert. En naturalytelse kan både falle bort, og komme tilbake. TilOgMed feltet trengs i de tilfellene der du får naturalytelsen tilbake ila permisjonen. Listen vi sender indikerer perioder der de er bortfalt